rosidl_default_generators/rosidl_default_generators-extras.cmake.in

```diff
@@ -1,15 +1,14 @@
 # generated from
 # rosidl_default_generators/rosidl_default_generators-extras.cmake.in
 
+find_package("ament_cmake_core" REQUIRED)
+ament_index_get_resources(rosidl_typesupport_c_packages "rosidl_typesupport_c")
+ament_index_get_resources(rosidl_typesupport_cpp_packages "rosidl_typesupport_cpp")
 set(_exported_dependencies
   "rosidl_typesupport_c"
   "rosidl_typesupport_cpp"
```

Comment:
going back to this. I think it's confusing to have some ament_index resources having the same name as some of our packages. Not to be addressed in this PR but I'd like to know if the overlapping names is on purpose and/or if we're open in changing that in the future

Reply:
I chose the name for the resource since I thought it makes sense to correlated them with the package. We can decide to rename them if we want to do that.